DXMainClient/DXGUI/Multiplayer/CnCNet/TunnelListBox.cs

```diff
@@ -63,6 +65,17 @@ public void SelectTunnel(string address)
             }
         }
 
+        /// <summary>
+        /// Gets whether or not a tunnel from the list with the given address is selected.
+        /// </summary>
+        /// <param name="address">The address of the tunnel server</param>
+        /// <returns>True if tunnel with given address is selected, otherwise false.</returns>
+        public bool IsTunnelSelected(string address)
+        {
+            int index = tunnelHandler.Tunnels.FindIndex(t => t.Address == address);
+            return index == SelectedIndex;
+        }
```

Comment:
Think the local variable is unnecessary here, could also use expression body here.